dev-util/meson-format-array/meson-format-array-0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{5,6,7} )
```

Comment:
Same here.

Reply:
The package has been very recently added.